packages/genesys-spark-components/.stylelintrc.json

```diff
@@ -1,3 +1,15 @@
 {
-  "extends": ["stylelint-config-genesys-spark-components"]
+  "extends": ["stylelint-config-genesys-spark-components"],
+  "overrides": [
+    {
+      "files": [
+        "src/components/stable/gux-button/**/*.{scss,html}",
+        "src/components/stable/gux-button-slot/**/*.{scss,html}"
```

Comment:
I think it would be safe to make this change in all components at once but to be cautious I will add this change to a few components at a time and make sure there is no unexpected issues.